content/en/docs/contributing/contributing-flow.md

```diff
@@ -118,6 +118,19 @@ Sometimes `/triage` is also added which helps us when following up Issues.
 * Kind indicates if it is a `bug` or a `feature` but also can be `documentation` or `cleanup` (general maintenance)
 * Priority is the priority it has for the cert-manager team, PRs are still very welcome for those!
 
+### Assignees meaning in PRs and issues
+
+Sometimes, you might see someone commenting with the
+[`/assign` prow command](https://prow.build-infra.jetstack.net/command-help#assign):
+
+```plain
+/assign @meyskens
+```
+
+We don't have a strong definition yet of what being an assignee means. On issues, it
+means "working on it". On PRs, well... it is meaningless. This is something we want to
```

Comment:
Follow up from this morning's standup:
issues: working on it
PRs: if author -> needs work to be done aka "changes requestes"
if empty -> needs review
if !author -> i am reviewing this

that is the pseudo code :D